### PR TITLE
Added fedora instruction + secondary muscle

### PR DESCRIPTION
**Pull request by alokhan**
Hi,

I thought the secondary muscle on the workout overview was missing so I added them.
I also added the package list needed on fedora 23 to get it running :)

Here is my first contribution to this very nice project !

### CODE REVIEW

File: wger/manager/models.py [hunk 1]
```diff
@@ -132,13 +134,19 @@ def canonical_representation(self):
...
+                        muscles_back_secondary.append(i)
 
                 day_canonical_repr.append(canonical_repr_day)
 
             workout_canonical_form = {'obj': self,
-                                      'muscles': {'front': muscles_front, 'back': muscles_back},
+                                      'muscles': {'front': muscles_front, 'back': muscles_back, 'frontsecondary':muscles_front_secondary, 'backsecondary': muscles_back_secondary },
                                       'day_list': day_canonical_repr}
-
+            print(workout_canonical_form)
```
Comment (rolandgeider) [inline wger/manager/models.py]: better remove this :)

**Comment by rolandgeider** [comment]
Thanks for the PR!!

You will need to update the tests, basically just adding the secondary muscles to the result. If you've got questions about this, just ask. Tests can be tedious to write but in the long run, they are really really useful.

I would suggest that you separate your PRs by feature, e.g. one for the fedora packages, one for the muscles. Then things can be discussed and changed independently.

**Comment by rolandgeider** [comment]
Oh, and don't forget to add yourself to the AUTHORS file :)

**Comment by alokhan** [comment]
Ok I will create two new separate pull request and I didn't see there was test until I saw them failing on travis :D I will fix them :)

**Comment by rolandgeider** [comment]
I mean, you can just leave this one, that was just meant as a suggestion for the future

**Comment by alokhan** [comment]
ok :P

**Comment by alokhan** [comment]
I have corrected the unit test but it seems that the build still fail

File: wger/manager/models.py [hunk 2]
```diff
@@ -132,13 +134,22 @@ def canonical_representation(self):
...
 
             workout_canonical_form = {'obj': self,
-                                      'muscles': {'front': muscles_front, 'back': muscles_back},
+                                      'muscles': {'front': muscles_front,
+                                                  'back': muscles_back,
+                                                  'frontsecondary': muscles_front_secondary,
+                                                  'backsecondary': muscles_back_secondary},
                                       'day_list': day_canonical_repr}
-
+            print(workout_canonical_form)
```
Comment (rolandgeider) [inline wger/manager/models.py]: You should remove this debug line

File: wger/manager/views/workout.py [hunk 3]
```diff
@@ -93,6 +93,13 @@ def view(request, pk):
         if i not in muscles_back:
             muscles_back.append('images/muscles/main/muscle-{0}.svg'.format(i))
 
+    for i in canonical['muscles']['frontsecondary']:
+        if i not in muscles_front:
```
Comment (rolandgeider) [inline wger/manager/views/workout.py]: The only thing am I'm not sure about is that if a workout has exercises that train a muscle as the main and the secondary muscle, the result is a bit fuzzy with the colors. I would suggest that we only colour the secondary muscles if there are no main ones, something like this:

```python
if i not in muscles_front and i not in canonical['muscles']['front']:
    [...]
```

The same below

**Comment by rolandgeider** [comment]
Hi @alokhan sorry for not answering earlier but I was on vacation and before leaving things were a bit crazy at work (I already told your friend Gary). In any case, I'm back and have taken a look at the PR, I have added two comments for small things to change, but then I can merge it :)

**Comment by alokhan** [comment]
all right no problem :)

**Comment by rolandgeider** [comment]
Alright, merged!

**Comment by alokhan** [comment]
hehe nice looking forward to do more like it :)

**Comment by rolandgeider** [comment]
:+1:

**Pull request merged**